### ISSUE TITLE
Wrong colorspace with recordings

### ISSUE THREAD

**Issue by ofrzeta** [signalwire/freeswitch#544]
I recently opened a bug report for Chrome because our recordings got rendered with wrong colors in Chrome. Now the Chrome team says they are doing it correctly and  "the color space is incorrect on the containers and frame" (https://bugs.chromium.org/p/chromium/issues/detail?id=1067377). 

I tried different settings in av.conf.xml and different versions of Freeswitch, also compiled 1.10.2 from Git on Debian 10 with no success. The resulting mp4/h264 does always show pxfmt/colorspace "yuvj420p(pc, gbr/unknown/unknown)" regardless of my settings in the above config. For instance I tried colorspace values 1 and 5 but it didn't make a difference.

**Comment by seven1240** [comment]
https://github.com/signalwire/freeswitch/commit/036d3e80857ad1414fb036ca9e6e6cb0a9d477f4#diff-f15644a7ddb5f203a3622948fc56d4a7 was the main change we support colorspace. can you find out what should we do to make it right? or make a pr ?

**Comment by seven1240** [comment]
you can try the test_avformat.c and see if you can make it correct by change to different color space. I'm not sure how to check it.

**Comment by ofrzeta** [comment]
After taking a look at the test I managed to get a conference recording with "bt709/unknown/unknown". For this I had to set the colorspace in the "AV Format Config" section:

```
<configuration name="avformat.conf" description="AVFormat Config">
  <settings>
      <param name="colorspace" value="1"/>
  </settings>
</configuration>
```

While setting the colorspace in the 'profile name="H264' didn't seem to have an effect.

**Comment by briankwest** [comment]
Did you set it in AV Codec too?

**Comment by ofrzeta** [comment]
Yes, we jave configured in in the H264 profile of the "AVCodec Config". I didn't yet check if it works without that setting.

**Comment by ofrzeta** [comment]
I finally tried this and I got the correct colorspace also with the colorspace settings in "AV Format Config" section only.
